trace/event.go

```diff
@@ -1,25 +1,33 @@
 package trace
 
 import (
+	"fmt"
 	"time"
-
-	"github.com/twinj/uuid"
 )
 
 const (
 	RequestReceived = iota
+	RequestCompleted
```

Comment:
I would create a new type for these and implement a `String()` method on them and have it return the name of the constant for its value. It helps tremendously with debugging and if you use `int` you can still use `iota` for initialization.